src/libertem/io/dataset/dm.py

```diff
@@ -22,6 +23,34 @@ def _get_offset(path):
     return offset
 
 
+class DMDatasetParams(MessageConverter):
+    SCHEMA = {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "$id": "http://libertem.org/DMDatasetParams.schema.json",
+      "title": "DMDatasetParams",
+      "type": "object",
+      "properties": {
+        "type": {"const": "DM"},
+        "path": {
+          "type": "string",
+        },
+        "scan_size": {
+            "type": "array",
+            "items": {"type": "number"},
+            "maxItems": 4,
+        },
+      },
+      "required": ["type", "path"]
+    }
+
+    def convert_to_python(self, raw_data):
+        data = {
+            "files": raw_data["path"].split(','),
```

Comment:
then we can also use another character for splitting, but then we need to add that character instead of ',' in frontend.

Reply:
Then what if the `path` contains this other character? I would suggest to instead use a list, then you don't have that problem at all. This may require some other changes, but would be the "proper" way to support opening multiple files.